File: answers/Python/@oseme-techguy/07-read-line-by-line.py
"""
    Solution to Read Line-by-Line
"""

if __name__ == '__main__':
    filepath = './answers/Python/@oseme-techguy/data/07-read-line-by-line.txt'
    with open(filepath, encoding='utf-8') as file:
        for line in file:
            print(line.strip('\n'))

File: programming/answers/C/@mykeels/07-read-line-by-line.c
#include <stdio.h>

int main() {
    FILE* fp;
    char buffer[255];

    fp = fopen("./answers/C/@mykeels/data/07-read-line-by-line.txt", "r");

    while(fgets(buffer, 255, (FILE*) fp)) {
        printf("%s", buffer);
    }

    fclose(fp);
}